schema/migration-2-0013-20200426.sql

```diff
@@ -0,0 +1,21 @@
+-- Persistent generated migration.
+
+CREATE FUNCTION migrate() RETURNS void AS $$
+DECLARE
+  next_version int ;
+BEGIN
+  SELECT stage_two + 1 INTO next_version FROM schema_version ;
+  IF next_version = 13 THEN
```

Comment:
This version will need to change. In fact, deleting this and running `create-migration` again should fix it.

Reply:
Not sure. If I rebase it then sure, otherwise not. Look at the state of https://github.com/input-output-hk/cardano-db-sync/tree/ksaric/CD-8/schema in the current branch now.

Reply:
Fixed.